### ISSUE TITLE
Vanadium tries to access storage through google play services

### ISSUE THREAD

**Issue by JoonasC** [GrapheneOS/Vanadium#192]
Sometimes(for example when viewing the Github 404 page), Vanadium will try to access the phone's storage through google play services. Ideally Vanadium shouldn't try to use google play services at all.

Screenshots:
![Screenshot_20220615-222520](https://user-images.githubusercontent.com/3898724/173909486-06781a8c-061b-47bf-98aa-a94ccd38c2e4.png)
![Screenshot_20220615-222555](https://user-images.githubusercontent.com/3898724/173909516-3c2baa64-a382-4af1-baa1-6c786cc67918.png)

**Comment by thestinger** [comment]
FIDO2 won't currently work because Vanadium isn't whitelisted by Play services as a browser allowed to use FIDO2 but we'll be implementing a workaround for it and we've tried to apply to get it whitelisted.

**Comment by JoonasC** [comment]
Ok, thanks for letting me know

**Comment by thestinger** [comment]
Also, to clarify, Play services thinks it needs the Media permission (calling it Storage is outdated) to use FIDO2. I don't think it's actually required. It's a false positive. We could report it and try to get them to remove the check.

**Comment by h-2** [comment]
> Chromium implements FIDO2 via a Play services library and has no fallback.

Would a different solution not make sense? I mean, it seems like FIDO is quite a security sensitive thing and entrusting that to blackbox Google libs may not be the best solution?

**Comment by thestinger** [comment]
https://github.com/GrapheneOS/Vanadium/issues/61

**Comment by geppi** [comment]
@thestinger 
> FIDO2 won't currently work because Vanadium isn't whitelisted by Play services as a browser allowed to use FIDO2 but we'll be implementing a workaround for it and we've tried to apply to get it whitelisted.

Is there some news in the meantime regarding your attempt to get Vanadium whitelisted?
I've tried to enroll my Pixel 6 as an additional security key on github via Vandium but it didn't work.
So I guess FIDO still doesn't work with Vanadium or maybe I did something wrong?

Also, did I understand this correctly that FIDO via sandboxed play services should work on GrapheneOS with other browsers that are already whitelisted? If yes, which ones should work? Is there a source to look this up?
